<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addT</class>
 <widget class="QDialog" name="addT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter destination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Transportation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
